docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-feature-integration.md

```diff
@@ -42,6 +42,7 @@ Make sure that the following modules have been installed:
 | MerchantUser | vendor/spryker/merchant-user |
 | MerchantUserGui |	vendor/spryker/merchant-user-gui |
 | MerchantStorage | vendor/spryker/merchant-storage |
+| MerchantWidget | vendor/spryker-shop/merchant-widget |
```

Comment:
- please add other widget modules from this feature
- SoldByMerchantWidget should be described in this IG, please check and remove from other guides
- ProductOfferSoldByMerchantWidget and ProductSoldByMerchantWidget don't exist anymore, please remove them from other IGs

Reply:
I added the MerchantWidget to the wrong section, it should be fine now. In total this feature contains two Widgets and both should be there now.